projects/plugins/jetpack/modules/simple-payments/simple-payments.php

```diff
@@ -36,31 +79,56 @@ static function getInstance() {
 		return self::$instance;
 	}
 
+	/**
+	 * Register scripts and styles.
+	 */
 	private function register_scripts_and_styles() {
 		/**
 		 * Paypal heavily discourages putting that script in your own server:
+		 *
 		 * @see https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/add-paypal-button/
 		 */
-		wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js', array(), null, true );
+		wp_register_script(
+			'paypal-checkout-js',
+			'https://www.paypalobjects.com/api/checkout.js',
+			array(),
+			self::$version,
```

Comment:
I would mimic what the widget (and what is elsewhere in Jetpack too) of using `JETPACK__VERSION`. Caches would refresh at least monthly, but require no human intervention (or error) if missed during a JS/CSS update.

The only issue is WP.com defines is as `wpcom`, so it would never bust cache. I think that's something we need to update on the WP.com side since a lot of code already uses that for versioning.

Reply:
Updated